### PR TITLE
Hide all the avatars but the first on small screens

### PR DESCRIPTION
**Pull request by LemonBoy**
Disclaimer, my {s,}css knowledge is shit but I tried anyway.
The idea is to do as discourse does and hide all but one circles in order not to waste precious screen space.

### CODE REVIEW

**Comment by dom96** [comment]
Good idea. Discourse actually makes sure that the avatar it shows is of the thread's author IIRC but we can implement that later.

File: public/css/nimforum.scss [hunk 1]
```diff
@@ -253,6 +253,18 @@ $threads-meta-color: #545d70;
...
 }
 
+.thread-avatar {
+  @extend .avatar;
+  @extend .avatar-sm;
+}
+
+// Hide all the avatars but the first on small screens.
+@media screen and (max-width: 600px) {
+  .thread-avatar:not(.first) {
```
Comment (dom96) [inline public/css/nimforum.scss]: You shouldn't need to introduce a new class for this, and definitely not two classes :)

You can use this: ``#threads-list .avatar:not(:first-child)`` (untested but should work).

Comment (LemonBoy) [reply public/css/nimforum.scss]: Done, I had to modify a bit the selector because each image is in a <a> element.

**Comment by dom96** [comment]
Thanks!

**Pull request merged**